### ISSUE TITLE
other TomoPy_GUI

### ISSUE THREAD

**Issue by decarlof** [bramgibs/TomoPy_GUI#3]
Very nice work, shall we consider merging with [TomoPy UI](https://github.com/tomography/ufot) or at least to share ideas/functionality? for example the ability to run in GUI mode or full command line or the ability the save the last run configuration status, I think are nice features to have on both

**Comment by bramgibs** [comment]
Absolutely, we should definitely share ideas/functionality at minimum,but I'm also willing to consider merging at some point. @newville suggested both the GUI/command option and batch config in person, but I haven't gotten around to working on those yet.

**Comment by decarlof** [comment]
one way to provide command line capability to a gui is using the config.py from [here](https://github.com/tomography/ufot/tree/master/ufot) and customize it to your paramentes

**Comment by bramgibs** [comment]
Thanks, I’ll give it a look and see what I can do.

**Comment by sandeep661** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://link.bnbreward.online/0mhPJ2)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://link.bnbreward.online/0mhPJ2) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @thesahinmolla, @frankiee12a9, @mrkvn, @dshovchko, @uniibu, @TiagoHA, @PartlyFluked, @Magic-Forces, @Kaushalop, @BaCoNL, @Kent55, @Twig6943, @shodgesio, @Lenglemetz, @sumudu-Jayasinghe, @TomCcYuan, @davidmerrique, @Loopios7, @yumyumfarted, @ogzgl, @josesenacosta, @Kasakinozomi1203, @LooJee, @endacyayisenga, @etanot, @ahapu, @swapnilmj, @Only1337, @valyakuttan, @willongithub, @than0s, @mattdanielbrown, @Robodog63, @dhassanali, @z0ccc, @jacobherrington, @uniqueitmail, @xsalimon333, @rvolosatovs, @Bensonlam168, @lfkdev, @valsaven, @mengopudding, @emirgoren, @luisbraganca, @zorky, @ytrinh, @jnbptstm, @JagdishUpadhyay, @janson403
